### ISSUE TITLE
Google SSO login fails (browser not supported)

### ISSUE THREAD

**Issue by shanepoints** [RocketChat/Rocket.Chat#22210]
I am trying to login using the OSX app.
<img width="300" style="display:inline-block;" src="https://user-images.githubusercontent.com/5091134/120393181-c0591280-c2ee-11eb-9633-33f2db5a70a0.png">
<img width="300" style="display:inline-block;" alt="Screen Shot 2021-06-01 at 3 30 52 PM" src="https://user-images.githubusercontent.com/5091134/120393023-87b93900-c2ee-11eb-8156-4dc72204b746.png">

The way I login to my account is with Google, so I've clicked on the "G" button at the login.
I am prompted to enter my email:
<img width="300" alt="Screen Shot 2021-06-01 at 3 40 02 PM" src="https://user-images.githubusercontent.com/5091134/120393761-b257c180-c2ef-11eb-8df3-05201a96fbaf.png">

After entering my email address and clicking "Next", the following error is shown:
<img width="300" alt="Screen Shot 2021-06-01 at 3 28 52 PM" src="https://user-images.githubusercontent.com/5091134/120393662-889e9a80-c2ef-11eb-9262-e56d24eb1c53.png">

**Comment by johncrisp** [comment]
Hi.

Please fill out a bug report properly. We can't help without this information.

https://github.com/RocketChat/Rocket.Chat/issues/new?assignees=&labels=&template=bug_report.md

Most importantly which site are you trying to contact?

Please test with https://open.rocket.chat

**Comment by johncrisp** [comment]
Hi and thanks for letting us know (you don't need to @ people!). Please we got you fixed!

> I sincerely tried to abide by the guidelines for reporting, but when I was creating the issue the only link I found was this one:

No problems. Now this is something I have been trying to understand for a while.

Can you please describe the process you followed to report your issue please? We gets lots of Issues where the reporter clearly never finds the New Issue/Template and I don't understand why.

Personally I go to a repo, goto Issues, and click New Issue which brings up the template. Perhaps people are dong this differently and we need to try and fix that.

I'd be grateful for your input.

I'll leave this open for the time being awaiting your comments.

**Comment by shanepoints** [comment]
Yeah, it's bizarre.  When I went to create a second issue, the UI seemed to be different than the first time.  I'm confident that what I did was this: 

1. Visit repo homepage and click on "issues":
<img width="587" alt="Screen Shot 2021-06-03 at 9 55 44 AM" src="https://user-images.githubusercontent.com/5091134/120675260-25794900-c452-11eb-964a-1dcc2c56bcb0.png">

2. Click on "new issue":
<img width="678" alt="Screen Shot 2021-06-03 at 9 56 21 AM" src="https://user-images.githubusercontent.com/5091134/120675285-2a3dfd00-c452-11eb-970c-42e520e56a82.png"

Like I said before, the first time I did this it simply opened a new blank issue.  The second time, I was greeted with the following choices:
<img width="1295" alt="Screen Shot 2021-06-03 at 9 59 49 AM" src="https://user-images.githubusercontent.com/5091134/120675613-7721d380-c452-11eb-9b21-a1f3749b5334.png">

**Comment by shanepoints** [comment]
To repro, I guess you would need a user account that has never filed an issue before?
